spec/index.html

```diff
@@ -1075,8 +1063,8 @@ <h3>Restricting the Value of Strings</h3>
           </div>
         </div>
         <p>The regular expression language is <a data-cite="XPATH-FUNCTIONS#regex-syntax">defined by XQuery
-            1.0 and XPath 2.0 Functions and Operators</a> and is based on <a data-cite=
-                                                                             "XMLSCHEMA-2#regexs">XML Schema Regular Expressions</a>.</p>
+            1and XPath Functions and Operators</a> and is based on
```

Comment:
could be
```suggestion
            and XPath Functions and Operators</a> and is based on
```

Reply:
or
```suggestion
            1 and XPath Functions and Operators</a> and is based on
```

Reply:
The former. Chnge done and searched for other occurances.